src/pages/EditMonitor.vue

```diff
@@ -1181,7 +1202,10 @@ export default {
         VueMultiselect,
         EditMonitorConditions,
     },
-
+    setup() {
+        const toast = useToast();
+        return { toast };
+    },
```

Comment:
Are you sure that this is nesssesary? Other functions also call spawn toasts without this already.